services-js/registry-certs/client/dao/CheckoutDao.test.ts

```diff
@@ -1,6 +1,5 @@
-// @flow
+import { FetchGraphql } from '@cityofboston/next-client-common';
```

Comment:
I know this change affected a lot of files, but I think this name change makes things much more clear!